content/blog/Adobe-silver-sponsor.md

```diff
@@ -0,0 +1,18 @@
+Title: Announcing Adobe as Silver Sponsor
+Date: 2017-10-18 10:32
+Category: 2017
+Slug: silver-sponsor-Adobe
+Summary: Silver Sponsor Adobe
+
+
+<img src="https://in.pycon.org/2017/images/sponsor/adobe.jpg">
+
+**[Adobe Systems Incorporated](http://www.adobe.com/in/)** is an American multinational computer software company. The company is headquartered in San Jose, California, United States. Adobe's focus has always been the creation of multimedia and creativity software products, with foray of rich Internet application software. Adobe tools enables to take productivity to the next level while preserving the creative vision. 
```

Comment:
Please make the following corrections
foray -> the foray
tools enables -> tools enable